client/src/components/Container.jsx

```diff
@@ -74,7 +90,17 @@ class Container extends React.Component {
     if (!this.state.showMore) {
       return (
         <div style={containerStyles.container}>
-          <h2 style={containerStyles.header}>Reviews</h2>
+          <div style={{display: 'flex'}}>
+            <h2 style={containerStyles.header}>Reviews</h2>
+            <div style={{alignSelf: 'center', marginLeft: 10}}>
+              {_.times(this.state.averageStars, (n) =>{
+                return (
+                  <img key={n} style={{width: '20px', height: '20px'}} src="https://s3-us-west-1.amazonaws.com/anstyicons/icon-star-512.png"></img>
```

Comment:
for the total store summary, Etsy renders half-stars.  It might be too late to incorporate that though, and not a huge deal.  The solution would be to use `.svg`s, but might take a while to get incorporated

Reply:
yeah I might try to come back and look into that. Thanks for letting me know